pkg/windows/eventwatcher/power_event_watcher_other.go

```diff
@@ -0,0 +1,27 @@
+//go:build !windows
+// +build !windows
+
+package eventwatcher
```

Comment:
feels weird to have this in a package called `windows/eventwatcher`. Also feels like "eventwatcher" is "powereventwatcher"

Reply:
Happy to rename whatever! I'll update the package to `powereventwatcher` so it's more consistent -- let me know if there's an update you want for this no-op one too.

Reply:
Nothing specific to the noop. Just pondering naming and locations.